api/src/main/java/jakarta/data/metamodel/package-info.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (c) 2023,2024 Contributors to the Eclipse Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * <p>A static metamodel for entities that are used in Jakarta Data repositories.</p>
+ *
+ * <p>The {@link StaticMetamodel} allows for type-safe operations that avoid the
+ * need to hard-code entity attribute names as Strings. For example,</p>
+ *
+ * <pre>
+ * &#64;Entity
+ * public class Product {
+ *     &#64;Id
+ *     public long id;
+ *
+ *     public String name;
+ *
+ *     public float price;
+ * }
+ *
+ * &#64;StaticMetamodel(Product.class)
+ * public class _Product {
+ *     SortableAttribute id = SortableAttribute.get();
+ *     TextAttribute name = TextAttribute.get();
+ *     SortableAttribute price = SortableAttribute.get();
+ * }
+ *
+ * ...
+ *
+ * &#64;Repository
+ * Product products;
```

Comment:
```suggestion
 * Products products;
```

Reply:
Thanks for catching this! Both you and Kyle spotted it.